### ISSUE TITLE
NullPointerException when rehydrating the snapshot repository

### ISSUE THREAD

**Issue by asibross** [codecentric/spring-boot-admin#1125]
Hey,
I am using a private build based on #3c4d6f96654ec2c26fc9d5f1bd81df4a5dfb68d5
Looks like the piece of code that threw the exception was introduced in https://github.com/codecentric/spring-boot-admin/commit/99d82c32df27a3f10be35e6ebcc1ef2d85cf385a

This is the exception I am seeing:
```
java.lang.NullPointerException: null
	at java.util.concurrent.ConcurrentHashMap.replace(ConcurrentHashMap.java:1556) ~[?:1.8.0_202]
	at de.codecentric.boot.admin.server.domain.entities.SnapshottingInstanceRepository.lambda$rehydrateSnapshot$3(SnapshottingInstanceRepository.java:88) ~[spring-boot-admin-server-2.1.3-sr.1-NetflixRelease.jar!/:2.1.3-sr.1-NetflixRelease]
	at reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.onNext(FluxMapFuseable.java:273) [reactor-core-3.2.6.RELEASE.jar!/:3.2.6.RELEASE]
	at reactor.core.publisher.FluxMapFuseable$MapFuseableConditionalSubscriber.onNext(FluxMapFuseable.java:287) [reactor-core-3.2.6.RELEASE.jar!/:3.2.6.RELEASE]
```
Did some debugging and it looks like the outdatedSnapshot is null.
![image](https://user-images.githubusercontent.com/11426802/54389652-d884de80-465d-11e9-818b-c0ede2daadd9.png)

**Comment by asibross** [comment]
I am assuming the above code should use the `outdatedSnapshots`:
`Instance outdatedSnapshot = this.oudatedSnapshots.get(id);`

**Comment by joshiste** [comment]
fixed on 2.1.x with fc22f77f0530a9efb5a109815d52ffd032f5cccc

**Comment by joshiste** [comment]
Thanks for spotting this mistake!
